src/aiAgentsConfiguration/aiAgentsConfigurationTreeDataProvider.ts

```diff
@@ -63,11 +65,13 @@ export class AIAgentsConfigurationTreeDataProvider implements VSCode.TreeDataPro
 
     const items: AIAgentsConfigurationItem[] = [];
     const isSupportingMCP = getCurrentAgentWithMCPSupport();
+    const agentWithHookSupport = getCurrentAgentWithHookSupport();
 
     const sonarQubeMCPServerConfigured = getCurrentSonarQubeMCPServerConfig() !== undefined;
     const rulesFileConfigured = await isSonarQubeRulesFileConfigured();
+    const hookScriptInstalled = agentWithHookSupport ? await isHookInstalled(agentWithHookSupport) : false;
 
-    if (!sonarQubeMCPServerConfigured && !rulesFileConfigured) {
+    if (!sonarQubeMCPServerConfigured && !rulesFileConfigured && !hookScriptInstalled && !agentWithHookSupport) {
```

Comment:
`hookScriptInstallled` already checks for `agentWithHookSupport`, no need to have it here as well